app/views/collection_drafts/new.html.erb

```diff
@@ -44,7 +44,7 @@
         <% end %>
       </div>
 
-      <%= render :partial => "drafts/forms/#{@draft_form}", :locals => { draft: @draft } %>
+      <%= render partial: "collection_drafts/forms/#{@draft_form}", :locals => { draft: get_resource } %>
```

Comment:
since you changed it from `:partial =>` to `partial:`, maybe you want to change `:locals =>` to `locals:` also?